src/features/rewards/tooltip/index.tsx

```diff
@@ -0,0 +1,60 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this file,
+* You can obtain one at http://mozilla.org/MPL/2.0/.
+*/
+
+import * as React from 'react'
+import {
+  StyledWrapper,
+  StyledTooltip,
+  StyledTooltipText,
+  StyledPointer
+} from './style'
+
+export interface Props {
+  id?: string
+  text?: string
```

Comment:
I think this should be `React.ReactNode` and not string

Reply:
done